src/TraceEvent/TraceEvent.csproj

```diff
@@ -41,7 +40,7 @@
     <WarningLevel>3</WarningLevel>
     <DocumentationFile>bin\Debug\Microsoft.Diagnostics.Tracing.TraceEvent.xml</DocumentationFile>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <NoWarn>0649 0618</NoWarn>
+    <NoWarn>0649,0618</NoWarn>
```

Comment:
I am not quite sure what you are proposing, but I am fine with the general principle.